bitcoind-rpc/src/main/scala/org/bitcoins/rpc/serializers/JsonWriters.scala

```diff
@@ -89,6 +89,11 @@ object JsonWriters {
     override def writes(addr: AddressType): JsValue = JsString(addr.toString)
   }
 
+  implicit object CurrencyUnitWrites extends Writes[CurrencyUnit] {
+    override def writes(amount: CurrencyUnit): JsValue =
+      JsNumber(amount.toBigDecimal)
```

Comment:
I don't think this is the default behavior we want, and this could be dangerous.

Reply:
Chiming in here: I think we shouldn't have a `Writes[CurrencyUnit]` implemented at all, in the same way that we shouldn't have a `Writes[FiniteDuration]`. Both of these types are unit agnostic (could be satoshis/bitcoins or milliseconds/hours), so we should explicitly convert them to the right unit at the call site;